lib/store/apiCallsStatus.ts

```diff
@@ -29,6 +29,10 @@ export const selectPatchApplicationStatus = (
   state: RootState
 ): ApiCallStatus | undefined => state.hrApiCallsStatus.patchApplication;
 
+export const selectSaveApplicationStatus = (
```

Comment:
Is this same as the existing `selectPatchApplicationStatus` selector? Perhaps we can drop one of them ?

Reply:
save is probably more appropriate than patch in the name